client/src/components/Loader/index.js

```diff
@@ -0,0 +1,3 @@
+import Loader from './Loader';
+
+export default Loader;
```

Comment:
That no write in 2 lines u may use reexport
export { default as Loader } from './Loader
and in file where u import Loader u need import as object { Loader }